hack/gen_values_yml.rb

```diff
@@ -0,0 +1,57 @@
+require "optparse"
+require "yaml"
+require_relative "../_plugins/lib"
+
+usage = "ruby hack/gen_values_yaml.rb <version> [arguments...]
+
+It's recommended to run this from the root of the Calico repository,
+as the default paths assume as much.
+
+<version> should be the major.minor version (e.g. v3.6) or master.
+
+--config    Path to the jekyll config. [default: _config.yml]
+--versions  Path to the versions.yml. [default: _data/versions.yml]
```

Comment:
Do we need more than just the versions yaml?

Do we want the ability to pick _which_ version to generate?

e.g. `--version master` versus `--version v3.6.0` ?